app/src/main/res/layout/item_susi_map.xml

```diff
@@ -30,7 +30,7 @@
 
                 <TextView
                     android:id="@+id/text"
-                    android:layout_width="wrap_content"
+                    android:layout_width="240dp"
```

Comment:
@iamareebjamal We are using custom layout for this, therefore `match_parent`is not working as excepted. Could you help me out in this.